hudi-utilities/src/test/java/org/apache/hudi/utilities/deser/TestKafkaAvroSchemaDeserializer.java

```diff
@@ -102,9 +99,10 @@ private IndexedRecord createExtendUserRecord() {
   public void testKafkaAvroSchemaDeserializer() {
     byte[] bytesOrigRecord;
     IndexedRecord avroRecord = createUserRecord();
-    SchemaTestProvider.schemaToReturn.set(origSchema);
-    KafkaAvroSchemaDeserializer avroDeserializer = new KafkaAvroSchemaDeserializer(schemaRegistry, new HashMap(defaultConfig));
-    avroDeserializer.configure(new HashMap(defaultConfig), false);
+    config.put("hoodie.deltastreamer.source.kafka.value.deserializer.schema", origSchema.toString());
```

Comment:
I ll replace this with`DataSourceWriteOptions.KAFKA_AVRO_VALUE_DESERIALIZER_SCHEMA()`

Reply:
Looking at this as I rebase #2833 against this PR.

Reply:
Alright thanks!